pkg/validator/cluster.go

```diff
@@ -23,18 +23,31 @@ import (
 
 	kdv1 "github.com/bluek8s/kubedirector/pkg/apis/kubedirector.bluedata.io/v1alpha1"
 	"github.com/bluek8s/kubedirector/pkg/catalog"
+	"github.com/bluek8s/kubedirector/pkg/observer"
 	"github.com/bluek8s/kubedirector/pkg/reconciler"
 	"github.com/bluek8s/kubedirector/pkg/shared"
 	"k8s.io/api/admission/v1beta1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 // membersPatchSpec is used to create the PATCH operation for adding a default
```

Comment:
Need to change the name of membersPatchSpec and membersPatchValue if this data structure is now being used for more than just patching the "members" value.

Reply:
Changed to clusterPatchSpec and clusterPatchValue